README.md

```diff
@@ -26,24 +26,30 @@ This application exposes the following endpoints:
 * Health endpoint: `<host>:<port>/health`
 * Web content: `<host>:<port>/<contextRoot>`
 * Web Application: `<host>:<port>/v1/example`
+* Swagger UI: `<host>:<port>/openapi/ui`
+* Openapi Document: `<host>:<port>/openapi`
 
 The web application has a health endpoint which is accessible at `<host>:<port>/health`. The ports are set in the `pom.xml` file.
 
 ## Steps
 
 You can [deploy this application to IBM Cloud](https://cloud.ibm.com/developer/appservice/starter-kits/java-liberty-app) or [build it locally](#building-locally) by cloning this repo first.  Once your app is live, you can access the `/health` endpoint to build out your cloud native application.
 
-### Deploying to IBM Cloud
+### Deploying 
+After you have created a new git repo from this git template, remember to rename the project. Edit the pom.xml and change the artifactId from the default name to the name you used to create the template.
 
-<p align="center">
-    <a href="https://cloud.ibm.com/developer/appservice/starter-kits/java-liberty-app">
-    <img src="https://cloud.ibm.com/devops/setup/deploy/button_x2.png" alt="Deploy to IBM Cloud">
-    </a>
-</p>
+Make sure you are logged into the IBM Cloud using the IBM Cloud CLI and have access to you development cluster. If you are using OpenShift make sure you have logged into the OpenShift CLI on the command line.
 
-Click **Deploy to IBM Cloud** to deploy this same application to IBM Cloud. This option creates a deployment pipeline, complete with a hosted GitLab project and a DevOps toolchain. You can deploy your app to Cloud Foundry, a Kubernetes cluster, or a Red Hat OpenShift cluster. OpenShift is available only through a standard cluster, which requires you to have a billable account.
+Install the IBM Garage for Cloud CLI.
 
-[IBM Cloud DevOps](https://www.ibm.com/cloud/devops) services provides toolchains as a set of tool integrations that support development, deployment, and operations tasks inside IBM Cloud.
+```$bash
+npm i -g @ibmgaragecloud/cloud-native-toolkit-cli
+```
+Use the IBM Garage for Cloud CLI to register the GIT Repo
+```$bash
+igc pipeline -n dev --tekton --pipeline ibm-appmod-liberty
+```
+See the **Deploy an app** guide in the [IBM Cloud-Native toolkit](https://cloudnativetoolkit.dev/) for details.
```

Comment:
Should clarify and say that this is under the "day 1" category at this URL. Possibly just say to go through the content under the entire "Day 1" category.